bazel/external/wasmedge.BUILD

```diff
@@ -0,0 +1,27 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+licenses(["notice"])  # Apache 2
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+)
+
+cmake(
+    name = "wasmedge_lib",
```

Comment:
It looks that WasmEdge build process tries to fetch and build Boost library if not found, which fails in Bazel.

If it's a required dependency, then I believe that you need to build Boost library in Bazel as well, and then add dependency on it to `wasmedge_lib`. Perhaps https://github.com/nelhage/rules_boost would work?

If it's not a required dependency, then you need to make sure that CMake won't attempt to fetch it during build process.

Reply:
Thanks, we noticed it and will fix it in the next release version.